src/northern_ireland/ni_staging.py

```diff
@@ -18,8 +18,7 @@ def read_ni_files(
 ) -> Tuple:
     """Read in CSV file and schema for NI data."""
     # read in csv file as pandas dataframes
-    paths = config[f"{config['global']['network_or_hdfs']}_paths"]
-    ni_full_response_file = paths["ni_full_responses_path"]
+    ni_full_response_file = config["ni_paths"]["ni_full_responses_path"]
```

Comment:
Sorry, I can't remember if you said NI should be functional or was to-do: this is raising an error.  ni_full_responses_path is under hdfs/network_paths, not ni_paths.

Reply:
yes, an error in the key, thanks.